### PR TITLE
Implemented tips

### PR DESCRIPTION
**Pull request by benapetr**

### CODE REVIEW

File: huggle/Resources/huggle_tips.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Huggle contains "EditBar" a little widget that contains both User and Page history in a neat way. It's not enabled by default though. If you are on a small screen, you can replace user and page history widgets with it!
```
Comment (Krenair) [inline huggle/Resources/huggle_tips.txt]: english

File: huggle/Resources/huggle_tips.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Most of preferences are stored on wiki in [[Special:MyPage/huggle3.css]], you can also change them there!
```
Comment (Krenair) [inline huggle/Resources/huggle_tips.txt]: Most preferences*

File: huggle/Resources/huggle_tips.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+Almost every widget on screen can be safely closed (except for navigation bar).
```
Comment (Krenair) [inline huggle/Resources/huggle_tips.txt]: for the*

File: huggle/preferences.cpp [hunk 4]
```diff
@@ -18,9 +18,11 @@
 #include "localization.hpp"
 #include "wikisite.hpp"
 #include "syslog.hpp"
+#include "resources.hpp"
 #include "mainwindow.hpp"
 #include "ui_preferences.h"
 #include <QMessageBox>
+#include <QLabel>
```
Comment (Krenair) [inline huggle/preferences.cpp]: Is that necessary?

File: huggle/resources.cpp [hunk 5]
```diff
@@ -58,6 +61,26 @@ QByteArray Huggle::Resources::GetResourceAsBinary(QString path)
...
+    {
+        QString tips = GetResource("/huggle/resources/Resources/huggle_tips.txt");
+        protips = tips.split("\n", ::QString::SkipEmptyParts);
+    }
+
+    if (protips.isEmpty() || protips.count() == 1)
+        return "Inform whoever packaged this Huggle that they did it wrong. There is no tips db.";
+
+    int random = GetRandom(0, protips.count() - 1);
+    while (last_tip != -1 && last_tip == random)
```
Comment (Krenair) [inline huggle/resources.cpp]: I would think the last_tip != -1 check is redundant? If it is false (i.e., last_tip is equal to -1) then the other clause (last_tip == random) should also be false

File: huggle/Resources/huggle_tips.txt (hunk 1 above)
Comment (benapetr) [inline huggle/Resources/huggle_tips.txt]: ?

Comment (Krenair) [inline huggle/Resources/huggle_tips.txt]: Huggle contains an "edit bar" - a little widget that contains both User and Page history in a neat way.

**Comment by Krenair** [comment]
what about the QLabel thing?

File: huggle/Resources/huggle_tips.txt [hunk 6]
```diff
@@ -0,0 +1,4 @@
+Huggle contains an "edit bar" a little widget that contains both User and Page history in a neat way. It's not enabled by default though. If you are on a small screen, you can replace user and page history widgets with it!
```
Comment (framawiki) [inline huggle/Resources/huggle_tips.txt]: Please add a "," after "edit bar"

File: huggle/Resources/huggle_tips.txt [hunk 7]
```diff
@@ -0,0 +1,4 @@
...
+Most preferences are stored on wiki in [[Special:MyPage/huggle3.css]], you can also change them there!
```
Comment (framawiki) [inline huggle/Resources/huggle_tips.txt]: Perhaps we can direcly show the username rather than a Special page ?

Comment (framawiki) [inline huggle/Resources/huggle_tips.txt]: *Manually* change them here

File: huggle/Resources/huggle_tips.txt [hunk 8]
```diff
@@ -0,0 +1,4 @@
...
+Almost every widget on screen can be safely closed (except for the navigation bar).
+Huggle supports tabs just like any other browser! If you are on some edit and you want to keep it for later, you can simply open new tab and continue checking the queue there. Queue is shared amongst all tabs.
```
Comment (framawiki) [inline huggle/Resources/huggle_tips.txt]: How to open a new tab ?

**Pull request merged**